### PR TITLE
fix waterch_test failed in windows

### PR DESCRIPTION
**Pull request by sniperking1234**
image
Unit test failed in Windows.

### CODE REVIEW

File: pkg/test/env/istio.go [hunk 1]
```diff
@@ -108,7 +108,11 @@ func getDefaultIstioTop() string {
 	if err != nil {
 		panic(err)
 	}
-	idx := strings.Index(current, "/src/istio.io/istio")
+	substr := "/src/istio.io/istio"
+	if runtime.GOOS == "windows" {
```
Comment (jwendell) [inline pkg/test/env/istio.go]: Can you do something with `path/filepath`? Perhaps using `Join()`?
https://golang.org/pkg/path/filepath/

Comment (sniperking1234) [reply pkg/test/env/istio.go]: @jwendell filepath used

**Pull request merged**